sdk/appcenter-data/src/main/java/com/microsoft/appcenter/data/models/LocalDocument.java

```diff
@@ -129,7 +131,10 @@ public long getOperationTime() {
      * @return whether the document is expired
      */
     public boolean isExpired() {
-        return getExpirationTime() <= System.currentTimeMillis();
+        if (mExpirationTime == TimeToLive.INFINITE){
```

Comment:
code is not formatted

Reply:
```suggestion
        if (mExpirationTime == TimeToLive.INFINITE) {
```

Reply:
Also this bug on infinite probably exists in 2.0.0, thus it should be in changelog.md

Reply:
sure :)